Files/Views/Pages/Properties.xaml.cs

```diff
@@ -146,13 +150,39 @@ private void Properties_Unloaded(object sender, RoutedEventArgs e)
             Unloaded -= Properties_Unloaded;
         }
 
-        private async void GetFolderSize(StorageFolder storageFolder)
+        private async void GetFolderSize(StorageFolder storageFolder, CancellationToken token)
         {
             var folders = storageFolder.CreateFileQuery(CommonFileQuery.OrderByName);
-            var fileSizeTasks = (await folders.GetFilesAsync()).Select(async file => (await file.GetBasicPropertiesAsync()).Size);
-            var sizes = await Task.WhenAll(fileSizeTasks);
-            var folderSize = sizes.Sum(singleSize => (long)singleSize);
+            var fileSizeTask = Task.Run(async () =>
+            {
+                var count = 0;
+                long size = 0;
+                uint index = 0;
+                try
+                {
+                    do
+                    {
+                        var files = await folders.GetFilesAsync(index, 16);
+                        count = files.Count;
+                        index += (uint)count;
+                        size += (await Task.WhenAll(files.Select(async file => (await file.GetBasicPropertiesAsync()).Size)))
```

Comment:
Is it possible to display current size value while calculating is not ended?

Reply:
Done